CONTRIBUTING.md

```diff
@@ -123,68 +125,92 @@ accepted.
+
+- The read-me files in the components are auto-generated. You won’t need to
+  update them. A big chunk of the documentation and prop-type validation is
+  happening via the `doc.js` files.
+- Code coverage and unit-testing is an important process of development.
+  A pre-commit hook exists which runs the test suite and aborts the commit if
+  any fail. To manually run tests, you should run `yarn test`. If you need to
+  update snapshots then run `yarn test-update`.
+- We are actively working on providing a seamless TypeScript experience. Don’t
+  forget to update corresponding `index.d.ts` files.
+- For code syntax alignment in your pull request, use [prettier].
+- Pull requests with code should include tests that validate the changes.
+
+We review issues and pull requests on a weekly basis (sometimes more
+frequently). If you feel we missed yours don’t hesitate to ping us on
+[Slack][slack community]!
+
+## Contributing to the Documentation
 
 Grommet uses an internal tool for most of its documentation. If you are looking
-to modify component to documentation then you only need to have a look at the
+to modify component documentation then you only need to have a look at the
 `doc.js` files.
 
-These files are used to generate the documentation on the grommet website. That
-code lives at the [grommet-site](https://github.com/grommet/grommet-site) repository.
+These files are used to generate the documentation on the Grommet website. That
+code lives in the [grommet-site] repository.
 
-Found an error in the documentation? [File an issue](https://github.com/grommet/grommet/issues).
+Found an error in the documentation? [File an issue][grommet issues].
 
-## Need more help?
+## Need More Help?
 
-Watch this [video](https://vimeo.com/129681048) to learn how to contribute to Grommet. The Github
-contribution workflow is somehow complex and we want to make sure we
-don't lose your contributions because of that.
-Note that the video is outdated and points to contributing on Grommet 1.X, but it may still be helpful for new users.
+Watch this [video] to learn how to contribute to Grommet. The GitHub
+contribution workflow is a bit complex and we want to make sure we don’t lose
+your valuable contributions because of that. Note that the video is outdated and
+talks about contributing to Grommet 1.X, but it may still be helpful for new
+users.
 
 ## References
 
-This contribution guide was inspired by the contribution guides for
-[Grunt](http://gruntjs.com/contributing),
-[CloudSlang](http://www.cloudslang.io/#/docs#contributing-code), and
-[Docker Library](https://github.com/docker-library/docs/tree/master/node).
-
-
+This contribution guide was inspired by the contribution guides for [Grunt],
+[CloudSlang], and [Docker Library].
+
+[cloudslang]: http://www.cloudslang.io/#/docs#contributing-code
+[design-kit]: https://github.com/grommet/design-kit
+[design-kit issues]: https://github.com/grommet/design-kit/issues
```

Comment:
I don't have a problem adding all these refs, but I do think the context of References isn't set for all of this content, and it might be overwhelming.

Reply:
I’m not sure I follow. What do you mean by “the context of References?” How might this content be overwhelming? (Note that these are basically meta details for Markdown so that it can properly link the reference style links mentioned earlier. This list of references is not rendered.)